packages/react-core/src/components/TextArea/TextArea.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/FormControls/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Additional classes added to the TextArea. */
+  className: PropTypes.string,
+  /** Flag to show if the TextArea is required. */
+  isRequired: PropTypes.bool,
+  /** Flag to show if the TextArea is valid or invalid. */
+  isValid: PropTypes.bool,
+  /** Value of the TextArea. */
+  value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  /** A callback for when the TextArea value changes. */
+  onChange: PropTypes.func,
+  /** Custom flag to show that the TextArea requires an associated id or aria-label. */
+  'aria-label': props => {
+    if (!props.id && !props['aria-label']) {
+      return new Error('TextArea requires either an id or label to be specified');
```

Comment:
can you update it to say aria-label instead of label here.
Can you also update the one error in the TextInput to match.